services/openshift/scripts/add_insecure_registry

```diff
@@ -36,7 +36,8 @@ function init_insecure_reg()
         sed -i.orig "/# INSECURE_REGISTRY=*/c\INSECURE_REGISTRY=\"\"" ${docker_conf_file}
         # Get machine IP address
         local local_ip=$(get_ip_address)
-        sed -i.back "s/INSECURE_REGISTRY=\"\(.*\)\"/INSECURE_REGISTRY=\"\1 --insecure-registry 172.30.0.0\/16 --insecure-registry $local_ip\"/" ${docker_conf_file}
+        local registry_host_name="hub.openshift.${local_ip}.xip.io"
```

Comment:
Actually, I am not so sure about this one. It does what we want, but why do you hardcode `xip.io`? The same [here](https://github.com/projectatomic/adb-utils/blob/master/services/openshift/scripts/openshift#L19) and here [here](https://github.com/projectatomic/adb-utils/blob/master/services/openshift/scripts/openshift_provision#L67). 

What if someone wants to go the Landrush route?

Is the problem not in [openshift_option](https://github.com/projectatomic/adb-utils/blob/master/services/openshift/openshift_option#L4). There we just use the hostname, but it should be something like `10.1.2.2.xip.io` or `$(hostname).10.1.2.2.xip.io`. In the latter case the hub host would be something like `hub.openshift.$(hostname).10.1.2.2.xip.io`.